### ISSUE TITLE
 Mapping "jk" to "<Esc>" in insert mode doesn't work after pressing "c"

### ISSUE THREAD

**Issue by He2hiwei** [VSCodeVim/Vim#3099]
**Describe the bug**
My `<Esc>` key is mapped to `jk` and it works fine in many cases but when I select some text and type `c` to change the selected text,  mapping `jk` to `<Esc>` in insert mode doesn't work.

**To Reproduce**

settings.json
```json
    "vim.insertModeKeyBindings": [
        {
            "before": ["j", "k"],
            "after": ["<Esc>"]
        }
    ]
```

1.  Select some text in normal mode
2.  type `c` to delete the selected text and enter insert mode
3.  type `jk`
4.  See error:  mapping `jk` to `<Esc>` in insert mode doesn't work
5.  If you type `<Esc>`, vim will exit insert mode and enter normal mode.

**Expected behavior**
Mapping `jk` to `<Esc>` in insert mode should work all the time.

**Environment (please complete the following information):**

I test this operation in both macOS and windows and see the same error.
- Extension (VsCodeVim) version:  0.16.6
- VSCode version: 1.27.2
- OS: macOS Mojave 10.14 / windows 7

**Comment by shawnaxsom** [comment]
This sounds like a duplicate of #3088. This was fixed with #3090, which has been merged and is waiting release.

**Comment by xconverge** [comment]
duplicate of #3088 

will try to get a release out tonight
